database/models.py

```diff
@@ -0,0 +1,28 @@
+from sqlalchemy import Column, Integer, String, DateTime, ForeignKey
+from sqlalchemy.orm import relationship
+from sqlalchemy.sql import func
+
+from database.engine import BaseSQLAlchemyModel
+
+
+class DBCity(BaseSQLAlchemyModel):
+    __tablename__ = "city"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(127), nullable=False, unique=True)
+    additional_info = Column(String(255))
+
+    temperature = relationship(
+        "DBTemperature", back_populates="city", uselist=False
+    )
```

Comment:
The `uselist=False` parameter in the `temperature` relationship suggests a one-to-one relationship between `DBCity` and `DBTemperature`. Ensure this is the intended design, as typically a city might have multiple temperature records over time. If a one-to-many relationship is intended, remove `uselist=False`.